lms/envs/common.py

```diff
@@ -219,6 +219,9 @@
     # Can be turned off if course lists need to be hidden. Effects views and templates.
     'COURSES_ARE_BROWSABLE': True,
 
+    # Can be turned off to disable the help link in the navbar
+    'ENABLE_HELP_LINK': True,
```

Comment:
Can you add the `.. toggle` preamble that other settings in this file use?

Reply:
Ah sorry I missed this email. I've added those to the file now :)